pom.xml

```diff
@@ -31,50 +27,158 @@
     <url>git@github.com:zanata/openprops.git</url>
   </scm>
 
-	<developers>
-		<!-- in alphabetical order (by id field) -->
-		<developer>
-			<id>sflaniga@redhat.com</id>
-			<name>Sean Flanigan</name>
-			<email>sflaniga@redhat.com</email>
-			<organization>Red Hat</organization>
-			<organizationUrl>http://www.redhat.com/</organizationUrl>
-			<timezone>10</timezone>
-		</developer>
-	</developers>
+  <developers>
+    <!-- in alphabetical order (by id field) -->
+    <developer>
+      <id>sflaniga@redhat.com</id>
+      <name>Sean Flanigan</name>
+      <email>sflaniga@redhat.com</email>
+      <organization>Red Hat</organization>
+      <organizationUrl>http://www.redhat.com/</organizationUrl>
+      <timezone>10</timezone>
+    </developer>
+  </developers>
+  <contributors>
+    <!-- in alphabetical order (by email field) -->
+    <contributor>
+      <name>Ding-Yi Chen</name>
+      <email>dchen@redhat.com</email>
+      <organization>Red Hat</organization>
+      <organizationUrl>http://www.redhat.com/</organizationUrl>
+      <roles>
+	<role>packager</role>
+	<role>tester</role>
+      </roles>
+      <timezone>10</timezone>
+    </contributor>
+  </contributors>
 
+  <properties>
+    <!--
+    Version of java compiler used for building. (Specified as major.minor,
+    as used by javac -source and -target).
+    -->
+    <required.java>1.8</required.java>
+    <!--
+      Version of Java used for runtime (major.minor) and for bytecode
+      generation. Should correspond with animal.sniffer.signature.
+    -->
+    <required.jvm>1.7</required.jvm>
```

Comment:
Unless you're using retrolambda, we can't compile on Java 1.8 without using a 1.8 JVM too.